README.md

```diff
@@ -29,3 +29,7 @@ tileGenerator(12)
   .on('data', function (tile) {
     console.log(tile);
   });
+
+### cli
```

Comment:
looks like the code block in final example of the current README wasn't closed properly. My bad!

If you could, add a triple ``` ` ``` on the line before your `### cli` header, then also frame your example code with triple ``` ` ``` as well so it's formatted nicely

Reply:
fixed